tests/components/deconz/test_services.py

```diff
@@ -73,7 +88,7 @@ async def test_service_unload(hass):
     ) as async_remove:
         await deconz.services.async_unload_services(hass)
         assert hass.data[deconz.services.DECONZ_SERVICES] is False
-        assert async_remove.call_count == 2
+        assert async_remove.call_count == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6259301)